lms/static/certificates/sass/_components.scss

```diff
@@ -537,7 +537,7 @@
 // #FOOTER
 // ------------------------------
 .footer-app {
-    border-top: rem(4) solid palette(grayscale, xx-trans);
+    border-top: rem(4) solid transparentize(palette(grayscale, base), 0.95);
```

Comment:
Can we use one of the five values of gray? I'm hesitant to do this kind of thing because then we'll end up with our same fifty shades of grey. It looks like you're trying to lighten our base grey a smidge? What about using the `accent`, or `back`?